Update PULL_REQUEST_TEMPLATE.md (#225)
magento/magento-cloud-docker: .github/PULL_REQUEST_TEMPLATE.md

1. ...
2. ...

### Release notes
<!---
    Please provide a list of public changes or documentation update pull requests
-->
1. ...
2. ...

> 

Reviewer: ```suggestion
### Release notes
<!---
If your PR includes user-facing changes, provide a meaningful release note. See https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html.
-->
<Add release note text>

### Associated documentation updates
<!--
 If your proposed update requires user documentation, submit a PR to the Magento DevDocs repository. For extensive updates requiring assistance, submit an issue to DevDocs. See https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md.
 -->
 <Add link to Magento DevDocs PR>

```

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose

>  - [ ] Pull request has meaningful release notes and documentation

Reviewer: ```suggestion
 - [ ] Pull request introduces user-facing changes and includes meaningful release notes and documentation
```

>  - [ ] Pull request has a meaningful releas notes and documentation

Reviewer 2: ```suggestion
 - [ ] Pull request has meaningful release notes and documentation
```

1. ...
2. ...

### Release notes

> <!---
> If your PR includes user-facing changes, provide a meaningful release note. See https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html.
> -->
> <Add release note text>

Reviewer: ```suggestion

For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud Docker release notes](https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html).
```

### Associated documentation updates
<!--
 If your proposed update requires user documentation, submit a PR to the Magento DevDocs repository. For extensive updates requiring assistance, submit an issue to DevDocs. See https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md.
 -->

>  <Add link to Magento DevDocs PR>

Reviewer: ```suggestion
<!--
 If your proposed update requires user documentation, submit a PR to the Magento DevDocs repository. For extensive updates requiring assistance, submit an issue to DevDocs. See https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md.
 -->
Add link to Magento DevDocs PR or Issue, if needed.
```
